source/targetfiltersource_test.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package source
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+
+	"sigs.k8s.io/external-dns/endpoint"
+)
+
+type mockTargetNetFilter struct {
+	targets map[string]bool
+}
+
+func NewMockTargetNetFilter(targets []string) endpoint.TargetFilterInterface {
+	targetMap := make(map[string]bool)
+	for _, target := range targets {
+		targetMap[target] = true
+	}
+	return &mockTargetNetFilter{targets: targetMap}
+}
+
+func (m *mockTargetNetFilter) Match(target string) bool {
+	return m.targets[target]
+}
+
+func (m *mockTargetNetFilter) IsConfigured() bool {
+	return true
+}
```

Comment:
This method is longer necessary.

Reply:
It's necessary to implement the interface(`endpoint.TargetFilterInterface`)

Reply:
I see what you mean, ✔️